src/components/Pickers/Suggestion/Suggestion.scss

```diff
@@ -0,0 +1,56 @@
+@import '../../../common/common';
+
+.ms-SearchMore-Button {
+  @include button-reset();
+  position: relative;
+  height: auto;
+  width: 100%;
+  padding: 0px 8px 0px 0px;
+  margin: 0;
+  text-align: left;
+
+  &:hover {
+      background-color: $ms-color-neutralLight;
+      cursor: pointer;
+  }
+
+  // TODO: Works in Chrome, but not working in IE
+  &:focus,
+  &:active {
+      background-color: $ms-color-themeLight;
+  }
+}
+
+.ms-Suggestions {
+  min-width: 180px;
+  .ms-Suggestions-Item {
+    height: auto;
+    width: 100%;
+    text-align: left;
+    border: none;
+    &:hover {
+      background: $ms-color-neutralLight;
+    }
+
+    &.is-suggested {
+      background: $ms-color-neutralQuaternary;
+      &:hover {
+        background: $ms-color-neutralQuaternaryAlt;
+      }
+    }
+  }
+  .ms-Suggestions-Title {
+    color: $ms-color-themePrimary;
+    font-size: $ms-font-size-s;
+    text-align: center;
+    text-transform: uppercase;
+    line-height: 40px;
+    border-bottom: 1px solid $ms-color-neutralLight;
+  }
+  .ms-Suggestion-Container {
```

Comment:
when you are making a component's css, class name guidance is:

.ms-TitleCasedClassName-camelCasedSection--kebab-cased-modifier

so this should be ms-Suggestion-container, and the title above should be ms-Suggestion-title (singular Suggestion?)